data/backend/authentication/admin.py

```diff
@@ -0,0 +1,51 @@
+from django.contrib import admin
```

Comment:
Side note, why is the `backend` app folder located under the `data` directory? I would naturally expect it to live at the root of the repo.